iina/Formatters.swift

```diff
@@ -23,3 +23,45 @@ class DecimalFormatter : NumberFormatter {
     return true
   }
 }
+
+class RestrictedNumberFormatter : NumberFormatter {
+
+  init(_ min: Float?, max: Float?, isDecimal: Bool) {
+    super.init()
+    if isDecimal {
+      self.numberStyle = .decimal
+    }
+    if let minValue = min {
+      minimum = minValue as NSNumber?
```

Comment:
> Also, we should consider using Double for min/max instead of Float.
That's wired. Both my local version and 'File Changed' on GitHub shows that I used `Double` instead of `Float`.